### DISCUSSION TITLE
Should the map legend be dynamic?

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1501] [Ideas]
Based on some comments on issue #356, there are some remaining questions:

1. Should the color legend (for want of a better term) be dynamic based on the current year?
2. Should the color legend be dynamic based on the visible zoom layer?
3. Is that color legend even useful at all?

This is a design question that might need user research.

**Comment by christoph-LSN** [comment]
1. No, the legend should show the min / max value over all years.
2. I don't know, becaus I don#t fully understand the zoom mechanisms
3. Yes.

Another question to me is how to compute the class breaks. In the chroma.js documentation [chroma.limits](https://gka.github.io/chroma.js/#chroma-limits) is described as a helper function. I tried 

limits: chroma.limits(data, 'e', 5) 
https://github.com/christoph-LSN/MT_Site/blob/3d49a5432c7c005a066de825a5aad1407b0190cf/_data/site_config.yml#L185

But there is no effect as you can see e.g. [here](https://christoph-lsn.github.io/MT_Site/1-1-1/)

@brockfanning @LucyGwilliamAdmin may be this is worth to start a new discussion ??
